wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/*
+Plugin Name:  AB Content Alterer
+Description:  WordPress Content Alterer: Change Unbounce and Leadpages in content to <strong>Instapage</strong>
+Version:      0.7
+Author:       andrzej.bialowas@instapage.com
+*/
+defined( 'ABSPATH' ) or die( 'No script kiddies please!' );
+add_action('the_content', 'abAlterContent');

```

Comment:
You are mixing coding styles. Decide whether you put spaces before/after braces or not